lib/src/widgets/message_view.dart

```diff
@@ -60,123 +58,138 @@ class MessageView extends StatefulWidget {
   final Color highlightColor;
   final bool shouldHighlight;
   final double highlightScale;
-  final ChatController chatController;
   final MessageConfiguration? messageConfig;
 
   @override
   State<MessageView> createState() => _MessageViewState();
 }
 
 class _MessageViewState extends State<MessageView>
-    with SingleTickerProviderStateMixin {
-  late AnimationController _animationController;
+    with TickerProviderStateMixin {
```

Comment:
Here is a specific for using TickerProviderMixin because the value of animation is being used at child level so it considers multiple animations that is why we have added TickerProviderMixin instead SingleTickerProviderStateMixin